wled00/wled.cpp

```diff
@@ -609,13 +609,13 @@ void WLED::initAP(bool resetAP)
     WLED_SET_AP_SSID();
     strcpy_P(apPass, PSTR(WLED_AP_PASS));
   }
+  #ifdef ARDUINO_ARCH_ESP32
+  WiFi.setTxPower(wifi_power_t(txPower));
+  #endif
   DEBUG_PRINT(F("Opening access point "));
   DEBUG_PRINTLN(apSSID);
   WiFi.softAPConfig(IPAddress(4, 3, 2, 1), IPAddress(4, 3, 2, 1), IPAddress(255, 255, 255, 0));
   WiFi.softAP(apSSID, apPass, apChannel, apHide);
-  #ifdef ARDUINO_ARCH_ESP32
-  WiFi.setTxPower(wifi_power_t(txPower));
```

Comment:
🤔 the "solution" basicially disables `WiFi.setTxPower(wifi_power_t(txPower))`, because trying to set TX power before AP is started will be ignored. I don't think this can be a general solution, because some boards require setting a reduced tx power otherwise wifi will not work.

Its interesting that this solves any problem ... maybe the default TX power is too high.